test/e2e-test-application/src/luigi-config/extended/userSettings.js

```diff
@@ -95,12 +96,6 @@ class UserSettings {
           options: ['red', 'green']
         }
       }
-    },
```

Comment:
This is needed. We have tests for displaying custom user settings micro frontend in general + on those nodes it's not necessary to have `settings` defined.